search.go

```diff
@@ -28,9 +28,9 @@ type TextField struct {
 
 type RangeField struct {
 	XMLName xml.Name
-	Is      float64 `xml:"is,omitempty"`
-	Min     float64 `xml:"min,omitempty"`
-	Max     float64 `xml:"max,omitempty"`
+	Is      interface{} `xml:"is,omitempty"`
+	Min     interface{} `xml:"min,omitempty"`
+	Max     interface{} `xml:"max,omitempty"`
 }
```

Comment:
To preserve concrete types, would it work to keep this as `float64`, and create a new struct `RangeDateField` with `Is`, `Min` and `Max` fields of type `time.Time`?